backend/src/main/java/de/neuefische/cgnjava222/productgallery/service/ProductService.java

```diff
@@ -14,8 +15,10 @@
 public class ProductService {
     private final ProductRepo productRepo;
 
-    public List<Product> getAllProducts() {
-        return productRepo.findAll();
+    public List<ProductListType> getAllProducts() {
+        return productRepo.findAll().stream().map(element ->
+                new ProductListType(element.id(), element.title(), element.pictureUrls().get(0), element.price())
```

Comment:
vllt. element in einer ähnlichen Namen umbenennen wie Product. Ist naheliegender